src/com/visualization/piechart/piechart.js

```diff
@@ -0,0 +1,69 @@
+import {h, Component} 				from 'preact/preact';
+
+import PieSegment					from 'com/visualization/piechart/piesegment';
+
+export default class PieChart extends Component {
+
+	constructor( props ) {
```

Comment:
constructor that just does super should not be needed

Reply:
It's a common stub. If something does ever need to be done during construction, it's already there waiting.

Reply:
but dead code... ;(

Reply:
If we're 100% sure it wont be used, then absolutely.